packages/desmos-std/Cargo.toml

```diff
@@ -16,3 +16,11 @@ serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 
 [dev-dependencies]
 cosmwasm-schema = { version = "1.0.0-beta" }
+
+[features]
```

Comment:
It would be useful to have default feature.
https://doc.rust-lang.org/cargo/reference/features.html#the-default-feature

Reply:
@dadamu added them